src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -1309,49 +1309,54 @@ namespace detail {
 CREATE_IS_CALLABLE(apply)
 CREATE_IS_CALLABLE_V(apply)
 
-template <typename TagsList>
-struct Apply;
+template <typename Func, typename... Args>
+constexpr void error_function_not_callable() noexcept {
+  static_assert(std::is_same_v<Func, void>,
+                "The function is not callable with the expected arguments.  "
+                "See the first template parameter for the function type and "
```

Comment:
`template parameter...` -> `template parameter of error_function_not_callable...`